### DISCUSSION TITLE
Unable to open X display

### DISCUSSION THREAD

**Discussion by attila-bulenda** [cypress-io/cypress#21262] [Questions and Help]
Hey folks,

I followed this guide here:
https://docs.cypress.io/guides/getting-started/installing-cypress#Windows-Subsystem-for-Linux

I followed all the steps, double checked it, but when I try to run the tests I get the following error message:

![image](https://user-images.githubusercontent.com/36601162/165930047-24497e28-83c4-4158-84a9-f0052ce9de7c.png)

Any ideas on how to fix this? I have bben Googling for hours now and can't find a proper solution.
Thanks in advance.

**Comment by thiagodiniz** [comment]
same here using WSL 2 Ubuntu with Windows 11
